src/main/java/seedu/phu/ui/HelpWindow.java

```diff
@@ -52,6 +52,8 @@ public class HelpWindow extends UiPart<Stage> {
             + "| Redo previous           |  √   | redo                                                              |\n"
             + "| undone command          |      |                                                                   |\n"
             + "+-------------------------+------+-------------------------------------------------------------------+\n"
+            + "| Exiting the program     |  x   | exit                                                              |\n"
```

Comment:
Can change to present tense to make it consistent?

Reply:
Sure!